wordpress-mobile/WordPress-Android
# Feature/move notification settings to notifications list

This PR moves Notifications Settings from Me screen to Notifications List.

I don't think there was a design for this one, so visuals are based on my assumption about how it should look like :)

[![Image from Gyazo](https://i.gyazo.com/4c5870dec2741802375eb131e9ce545d.gif)](https://gyazo.com/4c5870dec2741802375eb131e9ce545d)

To test:
- On the fresh app installation, login with a self-hosted site without Jetpack.
- Navigate to Notifications List.
- Notice Notifications Settings button is no there.
- Navigate to Me tab and login wot WordPress.com account.
- Navigate back to Notifications List.
- Notice that the Notifications Settings button is there and works!

Update release notes:
